src/main/kotlin/db/Db.kt

```diff
@@ -61,11 +62,20 @@ class Collection<K: Any, T: Any>(private val name: String, private val driver: D
 
     private fun key(id: K) = "$name/$id"
 
-    private fun fromString(str: String) = try {
-        gson.fromJson(str, type.java)
-    } catch (e: JsonSyntaxException) {
-        e.printStackTrace()
-        null
+    private fun fromString(str: String): T? {
+        var parsedObject = try {
+            gson.fromJson(str, type.java)
+        } catch (e: JsonSyntaxException) {
+            e.printStackTrace()
+            null
+        }
+        // У уже созданных аккаунтов по умолчанию должно быть true, а gson делает false
+        if (parsedObject is Account) {
```

Comment:
This is definitely out of place, which this check for a single type shows
Package `db` contains code of a general-purpose database. No project specific code is allowed here, especially object specific.